src/tools/devtools/web-performance.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Debugging performance for web apps
+description: Learn how to use Chrome DevTools to debug web performance issues.
+---
+
+{{site.alert.note}}
+  The following requires Flutter version 3.14 or later.
```

Comment:
Instead of the "The following", perhaps be a bit more specific:

```suggestion
  Profiling Flutter web apps requires Flutter version 3.14 or later.
```